Yoast/wpseo-woocommerce
# Add GTIN/MPN/ISBN functionality

## Summary

This PR can be summarized in the following changelog entry:

* Add a Yoast SEO section to the WooCommerce product data metabox, allowing you to enter your GTIN / ISBN / MPN number.

## Test instructions

This PR can be tested by following these steps:

* Open a product, see the new Yoast SEO section in the product data WooCommerce metabox, enter values there.
* See that those values are output on the frontend in the Schema.

Fixes #416 
Fixes https://github.com/Yoast/featurerequests/issues/15
